patternfly/patternfly-elements
# Cpfed 4318 fix jank on primary detail

## <Pull request description>

Fixing jank when it is in compact mode and user expands a section.

### Related issues

- https://issues.redhat.com/browse/CPFED-4318


### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-<pr_number>--happy-galileo-ea79c4.netlify.app/examples/) 


### Testing instructions

- [ ] Verify jank is gone
  1. Go to page with component, make sure it is in compact mode (it will have a one column layout)
  2. While the top level categories are visible (the left side of the component) expand one
  3. There shouldn't be any jank
  4. Go back by pressing heading, or escape on keyboard, make sure there isn't any jank here either (there wasn't before)


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
